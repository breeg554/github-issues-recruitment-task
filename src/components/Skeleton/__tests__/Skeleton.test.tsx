import { render } from "@testing-library/react";
import { Skeleton } from "..";
describe("Stars count component", () => {
  it("renders basic skeleton and match the snapshot", () => {
    const { container } = render(<Skeleton />);

    expect(container).toMatchSnapshot();
  });

  it("renders skeleton with custom props and match the snapshot", () => {
    const { container } = render(
      <Skeleton width="120px" height="50px" circle animate={false} />
    );

    expect(container).toMatchSnapshot();
  });
});
