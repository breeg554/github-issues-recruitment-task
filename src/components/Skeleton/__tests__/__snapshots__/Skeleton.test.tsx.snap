// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stars count component renders basic skeleton and match the snapshot 1`] = `
<div>
  <div
    class="sc-bdvvtL gxKZRD"
    color="#d7d7d7"
    height="150px"
    width="100%"
  />
</div>
`;

exports[`Stars count component renders skeleton with custom props and match the snapshot 1`] = `
<div>
  <div
    class="sc-bdvvtL btENuh"
    color="#d7d7d7"
    height="50px"
    width="120px"
  />
</div>
`;
